### ISSUE TITLE
Error downloading first video on an new install: no such table: main.media_fts

### ISSUE THREAD

**Issue by deldesir** [iiab/calibre-web#276]
![image](https://github.com/user-attachments/assets/1a5e749d-e681-468f-b96f-c5f6df63aae5)

The metadata fetch task failed with the following error:

```
root@box:/opt/iiab/iiab# tail -f /var/log/calibre-web.log
[2024-10-28 10:34:32,887]  INFO {cps.tasks.metadata_extract:155} Starting to fetch metadata for URL: https://www.youtube.com/watch?v=_vDZmVXtA7k
[2024-10-28 10:34:36,219] ERROR {cps.services.worker:202} no such table: main.media_fts
Traceback (most recent call last):
  File "/usr/local/calibre-web-py3/cps/services/worker.py", line 199, in start
    self.run(*args)
  File "/usr/local/calibre-web-py3/cps/tasks/metadata_extract.py", line 169, in run
    self._remove_shorts_from_db(conn)
  File "/usr/local/calibre-web-py3/cps/tasks/metadata_extract.py", line 59, in _remove_shorts_from_db
    conn.execute("DELETE FROM media WHERE path LIKE '%shorts%'")
sqlite3.OperationalError: no such table: main.media_fts
```

xklb and yt-dlp in use:
```
root@box:/opt/iiab/iiab# lb --version
3.0.016
root@box:/opt/iiab/iiab# yt-dlp --version
2024.10.22
```

**Comment by deldesir** [comment]
This bug does not affect dev version with the 8 PRs (from #244) applied as follow:
```
sudo -i
cd /usr/local/calibre-web-py3
git pull https://github.com/deldesir/calibre-web deldesir-search-captions --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-22 --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-23 --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-24 --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-25 --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-27 --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-xklb-schema --no-rebase --no-edit
git pull https://github.com/deldesir/calibre-web deldesir-patch-31 --no-rebase --no-edit
iiab-update -f
rm  /library/calibre-web/xklb-metadata.db

**Comment by holta** [comment]
@deldesir can you tell why Ed did not face this problem, when he tested PR iiab/iiab#3837 ("Revert PR #3815: python3-cryptography no longer needed") on Ubuntu 24.04 and Raspberry Pi OS about 24h ago?

**Comment by deldesir** [comment]
I cannot reproduce the issue on another install (RPI OS 64bit on RPI5  https://paste.centos.org/view/de8a6c97). I am thinking I might have gotten something wrong. Will try one more time before calling it quit.

**Comment by deldesir** [comment]
Closed as subsequent tests on Ubuntu 24.04 and 24.10* could not reproduce the issue.

---
* On Ubuntu 24.10, metadata fetching works but downloading doesn't as per #267
